Source/WebCore/Modules/mediasource/SourceBuffer.cpp

```diff
@@ -981,16 +981,26 @@ void SourceBuffer::evictCodedFrames(size_t newDataSize)
     if (isRemoved())
         return;
 
+    const auto hasTooManySamples = [&] {
+        const size_t evictionThreshold = m_private->platformEvictionThreshold();
+        if (!evictionThreshold)
```

Comment:
If there is no eviction threshold, which happens for non GStreamer code, this will bail out is false.

Reply:
it should proceed as normal If the eviction thershold is not set. could you please elaborate on issue you see here?

Reply:
I will below.